.style.yapf

```diff
@@ -0,0 +1,2 @@
+[style]
+based_on_style: yapf
```

Comment:
huh. is it really this simple a config file?

Reply:
Yes, I think our internal configs for Python formatting corresponded to the "chromium" config in  yapf and "chromium" was recently renamed to "yapf".